classes/rental.rb

```diff
@@ -0,0 +1,19 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] It would be better if we set up association between Rental and Book, Rental and Person. For example; rental belongs to book and also belongs to a person kind of association